.github/workflows/ci.yaml

```diff
@@ -54,6 +54,14 @@ jobs:
           working-directory: .
           args: --timeout=5m
 
+      - name: Clean up Docker resources
```

Comment:
What is this for? Do docker images persist across runs if we don't clear them out?